recipes/yosys/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "yosys" %}
+{% set version = "0.28" %}
+{% set sha256 = "36048ef3493ab43cfaac0bb89fa405715b22acd3927bf7fd3c4b25f8ad541c22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/YosysHQ/{{ name }}/archive/refs/tags/yosys-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('yosys', max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config 
+  host:
+    - make
+    - bison
+    - boost
+    - flex
+    - graphviz
+    - libffi
+    - libzlib
+    - python
+    - readline
+    - tk
+  run:
+    - libffi
+    - libzlib
+    - python
```

Comment:
There should probably be a minimum version requirement for your python dependency.

Reply:
"Non noarch packages should have python requirement without any version constraints."

Reply:
I forget if there's a different noarch I should be using, this is not a Python package

Reply:
`noarch: generic` is a thing, but it implies that the exact same artifacts can be used on *any* platform. Since you are using c compilers, I doubt that this applies.

Reply:
Linter objects to it anywhere, will have to leave it

Reply:
The linter is just script that checks for patterns. It can't force you to do anything. 😄